.werft/util/kubectl.ts

```diff
@@ -180,3 +180,7 @@ export function findFreeHostPorts(pathToKubeConfig: string, ranges: PortRange[],
     }
     return results;
 }
+
+export function waitForDeploymentToSucceed(pathToKubeConfig: string, name: string, namespace: string, type: string) {
+    exec(`export KUBECONFIG=${pathToKubeConfig} && kubectl rollout status ${type} ${name} -n ${namespace}`)
```

Comment:
```suggestion
    exec(`KUBECONFIG=${pathToKubeConfig} kubectl rollout status ${type} ${name} -n ${namespace}`)
```

should be sufficient and has the advantage that it does not permanently set `KUBECONFIG`

Reply:
This sounds like a very good suggestion. I will raise a separate PR to make this change as we have this in a lot of places.